docs/contributing.md

```diff
@@ -3,6 +3,23 @@
 We'd love to accept your patches and contributions to this project. There are
 just a few small guidelines you need to follow.
 
+## Getting started
+
+Currently, you need to install Python 3.6 for developing Flax, and `svn` for running the `run_all_tests.sh` script. After installing these prerequisites, you can clone the repository, set up your local environment, and run all tests with the following commands:
+
+```
+git clone https://github.com/google/flax
+cd flax
+python3.6 -m virtualenv env
+. env/bin/activate
+pip install -e . .[testing]
+./tests/run_all_tests.sh
+```
+
+Alternatively, you can also develop inside a Docker container : See [`dev/README.md`](https://github.com/google/flax/blob/master/dev/README.md).
+
+We welcome pull requests, in particular for those issues [marked as PR-ready](https://github.com/google/flax/issues?q=is%3Aopen+is%3Aissue+label%3A%22Status%3A+pull+requests+welcome%22). For other proposals, we ask that you first open an Issue to discuss your planned contribution.
```

Comment:
(I am just skimming right now. But let's point people to GitHub discussions rather than issues to propose new contributions or feature requests)